tests/Unit/Subscriber/Core/RegistratorSubscriberTest.php

```diff
@@ -104,6 +140,17 @@ public function testNodeFromRegistry()
         $this->subscriber->handleNodeFromRegistry($this->persistEvent->reveal());
     }
 
+    public function testReorderNodeFomRegistry()
```

Comment:
The human language description has the potential to be much more
preceise and expressive than a method name. This is how all the tests in
this library have been written.

On Mon, Jul 20, 2015 at 02:10:34PM -0700, Daniel Rotter wrote:
>    In [1]tests/Unit/Subscriber/Core/RegistratorSubscriberTest.php:
> 
>  > @@ -104,6 +140,17 @@ public function testNodeFromRegistry()
>  >          $this->subscriber->handleNodeFromRegistry($this->persistEvent->reveal());
>  >      }
>  >
>  > +    public function testReorderNodeFomRegistry()
> 
>    As already said I am against adding these comments, most of the time they
>    are just duplicating information. The name already shows that it is
>    testing if the node is set from the registry on the reorder event. If you
>    think the name is not sufficient I would still prefer to change the name,
>    instead of adding more comments.
> 
>    —
>    Reply to this email directly or [2]view it on GitHub.
> 
>    Reverse link: [3]unknown
> 
> References
> 
>    Visible links
>    1. https://github.com/sulu-io/sulu-document-manager/pull/28#discussion_r35044530
>    2. https://github.com/sulu-io/sulu-document-manager/pull/28/files#r35044530
>    3. https://github.com/sulu-io/sulu-document-manager/pull/28/files#r35044530